cyences_app_for_splunk/bin/cs_product_list.py

```diff
@@ -285,7 +285,7 @@ def build_app_dependency_search(app_list):
                 "macro_name": "cs_azure",
                 "label": "Azure Active Directory Data",
                 "search_by": "sourcetype",
-                "search_values": "azure:aad:audit",
+                "search_values": "azure:aad:audit,azure:aad:signin",
```

Comment:
I think we need to update the Document as well to show what data we need from this Add-on.